pkg/apis/v1/nodepool_status.go

```diff
@@ -27,6 +27,8 @@ const (
 	ConditionTypeValidationSucceeded = "ValidationSucceeded"
 	// ConditionTypeNodeClassReady = "NodeClassReady" condition indicates that underlying nodeClass was resolved and is reporting as Ready
 	ConditionTypeNodeClassReady = "NodeClassReady"
+	// ConditionTypeUnhealthy = "Unhealthy" condition indicates when the nodepool has more 20% of nodes in an unhealthy state
+	ConditionTypeUnhealthy = "Unhealthy"
```

Comment:
Can we mark this status condition as alpha and only add it when we have enabled the auto-repair feature?

Reply:
Should this be unhealthy or should this be a condition `Healthy` that can be marked as `False`?

Reply:
I added a note in a comment above , and should only be marked on the nodepool if node repair is enabled